staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -513,6 +515,9 @@ func InClusterConfig() (*Config, error) {
 		tokenFile  = "/var/run/secrets/kubernetes.io/serviceaccount/token"
 		rootCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 	)
+	// TODO: the context parameter can be added to the method in the future
```

Comment:
The goal is to convert the code *now*, not later... :hot_face: 

I don't know what the right API should be. That is what we need to find out.